internal/refine/refine.go

```diff
@@ -0,0 +1,52 @@
+package refine
+
+import (
+	"unicode/utf8"
+
+	"github.com/Decentr-net/cerberus/internal/schema"
+)
+
+// Cookie refines the given cookie is not autogenerated
+func Cookie(cookie *schema.V1Cookie) bool {
```

Comment:
Yes, Cookies and SearchHistory are the same at the moment, but it is likely to be changed.